packages/carbon-graphs/tests/unit/controls/Scatter/ScatterLoad-spec.js

```diff
@@ -470,115 +475,6 @@ describe('Scatter - Load', () => {
         });
       });
     });
-    describe('When values are non-contiguous', () => {
```

Comment:
Are these tests are not valid with the current changes since we are throwing an error?

Reply:
Yes Veera, since our way of handling null values changed, those tests are not valid anymore.